zarr/tests/test_convenience.py

```diff
@@ -415,15 +571,33 @@ def temp_h5f():
 
 
 @pytest.mark.skipif(not have_h5py, reason='h5py not installed')
-def test_copy_h5py_to_zarr():
-    _test_copy(temp_h5f, group)
+class TestCopyHDF5ToZarr(TestCopy):
+
+    def __init__(self, *args, **kwargs):
+        super(TestCopyHDF5ToZarr, self).__init__(*args, **kwargs)
+        self.source_h5py = True
+        self.dest_h5py = False
+        self.new_source = temp_h5f
+        self.new_dest = group
 
 
 @pytest.mark.skipif(not have_h5py, reason='h5py not installed')
-def test_copy_zarr_to_h5py():
-    _test_copy(group, temp_h5f)
+class TestCopyZarrToHDF5(TestCopy):
+
+    def __init__(self, *args, **kwargs):
+        super(TestCopyZarrToHDF5, self).__init__(*args, **kwargs)
+        self.source_h5py = False
+        self.dest_h5py = True
+        self.new_source = group
+        self.new_dest = temp_h5f
 
 
 @pytest.mark.skipif(not have_h5py, reason='h5py not installed')
-def test_copy_h5py_to_h5py():
-    _test_copy(temp_h5f, temp_h5f)
+class TestCopyHDF5ToHDF5(TestCopy):
+
+    def __init__(self, *args, **kwargs):
+        super(TestCopyHDF5ToHDF5, self).__init__(*args, **kwargs)
+        self.source_h5py = True
+        self.dest_h5py = True
+        self.new_source = temp_h5f
+        self.new_dest = temp_h5f
```

Comment:
Sorry if I overlooked it, but was there a ZarrToZarr case?

Reply:
Yes, sorry, it's the super-class, i.e., TestCopy implements zarr-to-zarr. I could make this more obvious.

Reply:
Thanks.